config/initializers/rack_rewrite.rb

```diff
@@ -2,7 +2,7 @@
 
 # URL for Nuts Server deployed on Heroku, redirects download/update requests to latest release
 # on GitHub Releases with platform parameters
-RELEASE_SERVER_URL = "https://classroom-assistant-releases.herokuapp.com"
+RELEASE_SERVER_URL = "http://classroom-assistant-releases.herokuapp.com"
```

Comment:
Why are we making this change?

Reply:
Oopps the merge of master brought that back